docs/source/api/accumulating_functions.rst

```diff
@@ -12,13 +12,9 @@ Accumulating functions
 Defined in ``xtensor/xmath.hpp``
 
 .. doxygenfunction:: cumsum(E&&)
-   :project: xtensor
```

Comment:
Why removing this? Is it a deprecated feature now?

Reply:
Indeed